components/user_settings/display/user_settings_display.jsx

```diff
@@ -682,6 +682,7 @@ export default class UserSettingsDisplay extends React.Component {
                         setRequireConfirm={this.props.setRequireConfirm}
                         setEnforceFocus={this.props.setEnforceFocus}
                         allowCustomThemes={this.props.allowCustomThemes}
+                        focused={this.props.prevActiveSection === this.prevSections.theme}
```

Comment:
Can you elaborate on why special casing is required?

Reply:
Because `ThemeSetting` is a sub-component instead of `SettingItemMin` in other cases.